ResourceStatusSystem/scripts/dirac-rss-sync.py

```diff
@@ -127,10 +135,9 @@ def initSites():
     Initializes Sites statuses taking their values from the "SiteMask" table of "JobDB" database.
   '''
 
-  jobClient = JobDB()
-  rssClient = ResourceStatusDB()
+  rssClient = ResourceStatusClient.ResourceStatusClient()
```

Comment:
`ResourceStatusClient.ResourceStatusClient()` does not contain ResourceStatusDB's `addIfNotThere` function which is used after a few lines [here](https://github.com/atsareg/DIRAC/blob/32a6f5f310d42203e65bed28ded69defdc467998/ResourceStatusSystem/scripts/dirac-rss-sync.py#L151)